src/components/Autosuggest/Autosuggest.js

```diff
@@ -185,12 +185,8 @@ export class Autosuggest extends React.Component {
 
```

Comment:
<img width="1101" alt="Screenshot 2020-07-07 at 11 38 45" src="https://user-images.githubusercontent.com/19436304/86765778-ebd56d80-c049-11ea-8f00-d3b3d5590360.png">
After some manipulation with `LocationSelector` console has an error.

Reply:
I created a separate issue for this story.
https://jira.textkernel.nl/browse/ONEUI-198